src/js/views/forms/form/form_views.js

```diff
@@ -169,12 +176,31 @@ const LayoutView = View.extend({
       replaceElement: false,
     },
     status: '[data-status-region]',
+    widgets: '[data-widgets-region]',
+  },
+  templateContext() {
+    return {
+      formHasWidgets: this.formHasWidgets,
+    };
+  },
+  initialize({ patient, widgets }) {
```

Comment:
Nothing is using the attached patient and only the `templateContext` will use `widgets` so it could probably just be a `this.getOption` in `templateContext` or we should convert the other `getOption`s. There's no hard and fast rule as to when to use `getOption` over attaching at `initialize`. They should always produce the same results.. So it's sort of a judgement call to whether a few `getOption`s are cleaner than an initialize.  I typically use initialize if more than a couple methods will be using the option.  But generally if we attach something we shouldn't use `getOption` as well.

Reply:
I'll use the `getOption()` method to keep consistent with what's used elsewhere in the file. Don't have a preference to either one. Would rather be consistent.